schemas.md

```diff
@@ -29,6 +29,31 @@ The activity of watching a video
 - `mixin:previews`
 - `url`: The URL of the video
 
+### activity.watching.movie
+
+The activity of watching a movie
```

Comment:
This is a duplicate of `activity.browsing.video`. At least the schema should follow a similar convention.

- `title`: Video title
- `description`: Video description
- `url`: The URL of the video

I would not include foreign database IDs here. If they were to be included, I'd create a separate label for each (`imdb_id`, `tmdb_id` etc).

Reply:
It's fine if you rename activity.browsing.video to activity.watching.video. Just make sure to create a migration for the existing entries.

Reply:
It is a bit of a duplicate. I hope the new changes are more in line with what you would like

At the moment my instinct is to leave activity.watching.video as it is.